frontend/src/pages/Admin/UpdateQuizPage.jsx

```
@@ -0,0 +1,136 @@
+	const [loading, setLoading] = useState(true);
+	const [error, setError] = useState(null);
+	const [quiz, setQuiz] = useState({
+		name: "",
+		description: "",
+	});
+
+	useEffect(() => {
+		const fetchQuiz = async () => {
+			try {
+				const categories = await getAllCategories();
+				const category = categories.find((c) => c.id === parseInt(categoryId));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)